client/finality-grandpa/src/tests.rs

```diff
@@ -94,6 +94,15 @@ impl TestNetFactory for GrandpaTestNet {
 		ProtocolConfig::default()
 	}
 
+	fn add_full_peer(&mut self) {
```

Comment:
For any other reviewers, this function is called [here](https://github.com/paritytech/substrate/blob/master/client/finality-grandpa/src/tests.rs#L74) and shadows [`add_full_peer` here](https://github.com/paritytech/substrate/blob/master/client/network/test/src/lib.rs#L617).